tools/bcl-test-importer/BCLTestImporter/BCLTestProjectGenerator.cs

```diff
@@ -390,6 +389,16 @@ internal static IEnumerable<string> GetIgnoreFiles (string templatePath, string
 			}
 		}
 
+		internal static string GetProjectGuid (string projectName) 
+		{
+			//use MD5 hash to get a 16-byte hash of the string: 
+			using (var provider = new MD5CryptoServiceProvider ()) {
```

Comment:
minor (since it's not in the platform assembly) but `MD5.Create ()` is better (since it will pick up the best implementation of MD5 available).